### PR TITLE
Develop ci

### PR DESCRIPTION
**Pull request by kfhoech**
Reconfigure Tycho/Maven build and add integration with Travis-CI.

### CODE REVIEW

**Comment by kfhoech** [comment]
Yay! Travis-CI checks pass!

File: tools/case/json.plugin/.project [hunk 1]
```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <projectDescription>
-	<name>json.plugin</name>
+	<name>com.collins.fmw.cyres.json.plugin</name>
```
Comment (tlogan) [inline tools/case/json.plugin/.project:3]: Could we change the name of the "case" directory to "staircase" and include that in the prefix of all the java packages in that directory; e.g. "com.collins.fmw.staircase.json.plugin" ?

File: tools/case/architecture.plugin/.project [hunk 2]
```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <projectDescription>
-	<name>architecture.plugin</name>
+	<name>com.collins.fmw.cyres.architecture.plugin</name>
```
Comment (tlogan) [inline tools/case/architecture.plugin/.project:3]: Could we rename package prefix "cyres" to "staircase" and "case" directory to "staircase"?

**Review by tlogan — APPROVED** [review]
These changes look fine to me.  I think we should rename a few things, either in this commit or the next.

- Could we use the same prefix/package conventions for agree, resolute etc and the new staircase tools?
- Could we rename "case" directory to "staircase" and "cyres" to "staircase"?
- Do you have any ideas for a better name than "formal-methods-workbench"?  Something shorter would be nice.

File: tools/case/json.plugin/.project (hunk 1 above)
Comment (kfhoech) [reply tools/case/json.plugin/.project:3]: Let's leave this pull request as it is.  As this has to do with "branding" of our tools, please discuss with the project PI (Darren) as to how it should be done.

File: tools/case/architecture.plugin/.project (hunk 2 above)
Comment (kfhoech) [reply tools/case/architecture.plugin/.project:3]: Let's leave this pull request as it is.  As this has to do with "branding" of our tools, please discuss with the project PI (Darren) as to how it should be done.

**Comment by kfhoech** [comment]
I'm going to continue on this branch to add an update site (accumulating released versions) and the product build.

**Pull request merged**